gym/lib/gym/runner.rb

```diff
@@ -97,7 +97,7 @@ def build_app
                                                 ErrorHandler.handle_build_error(output)
                                               end)
 
-      mark_archive_as_built_by_gym(BuildCommandGenerator.archive_path)
+      mark_archive_as_built_by_gym(BuildCommandGenerator.archive_path) unless Gym.config[:skip_archive]
       UI.success("Successfully stored the archive. You can find it in the Xcode Organizer.") unless Gym.config[:archive_path].nil?
       UI.verbose("Stored the archive in: " + BuildCommandGenerator.archive_path)
```

Comment:
We should probably wrap all these lines if we skip archive since there won't be anything in that path 😇 

```rb
if Gym.config[:skip_archive]  
  mark_archive_as_built_by_gym(BuildCommandGenerator.archive_path)
  UI.success("Successfully stored the archive. You can find it in the Xcode Organizer.") unless Gym.config[:archive_path].nil?
  UI.verbose("Stored the archive in: " + BuildCommandGenerator.archive_path)
end
```

Reply:
Ok I will wrap these lines although with `if !Gym.config[:skip_archive]`

Reply:
@Skoti I think we still wants something like this ☝️ We don't want any archive path stuff printed out because it didn't make one, correct?

Reply:
correct, that is why I think your example should use "!" in the if statement 😃